src/main/webapp/app/entities/modeling-exercise/modeling-exercise-update.component.ts

```diff
@@ -114,14 +115,22 @@ export class ModelingExerciseDialogComponent implements OnInit {
         );
     }
 
+    previousState() {
```

Comment:
Please also add documentation in this component.

Reply:
Added